roles/openshift-applier/tasks/pre-check.yml

```diff
@@ -0,0 +1,16 @@
+---
+- name: Exit if ansible version doesn't meet minimum requirements
+  fail:
+    msg: "openshift-applier requires at least Ansible 2.5 in order to proceed"
+  when:
+  - "ansible_version.full is version('2.5','<')"
+
+- name: test
+  shell: "oc version | head -1 | awk '{print $2}' | awk -F'+' '{print $1}'"
+  register: unknown_param_check
+
+- name: "Set Fact"
+  set_fact:
+    oc_ignore_unknown_parameters: false
+  when:
+    "unknown_param_check.stdout[1:] is version('3.6','<=')"
```

Comment:
Looks to be missing the "leading" `-` for the when list (we typically use lists) ... also, not sure there's a need to use a subscript for stdout as it should just be a single line output anyway. Lastly, what happens if no valid output is generated? Will it assume a version newer than 3.6? I.e.: default to false? If not, may want to safeguard to ensure it defaults to false and doesn't error out in case no stdout is produced or doesn't contain valid values.